### ISSUE TITLE
Pdfs in images from Wikimedia

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#2421]
## Description

We have some pdf file data in the image table. It passes the dead link validation, but shows up as "Image Not Available" in the frontend.

If cleaning up the catalog is quick, we should do that. Then these items will be fixed after the image refresh.

I'm not sure if we want to also add a hot-fix in the API layer to prevent files with `.pdf` in the URL from showing up in the results. If we added a file type check to the dead link filter, we would stop sending unnecessary requests to the source. The status is currently "200" because the file _is_ available, even though it doesn't show up in the UI.

## Reproduction

See: https://api.openverse.engineering/v1/images/dffcffbe-3980-4cc7-bc61-804851cb9121/
and https://openverse.org/image/dffcffbe-3980-4cc7-bc61-804851cb9121

## Additional context

@cc @krysal, I think you have done a similar cleanup in the catalog, haven't you?

**Comment by krysal** [comment]
> @krysal, I think you have done a similar cleanup in the catalog, haven't you?

Yes, to remove audio files from the image table. But this is an interesting case, the link actually does return an image. It opens a debate: should we consider accepting PDF images or just remove them? Can Wikimedia provide a different format more traditionally accepted as an image?

**Comment by AetherUnbound** [comment]
Thanks for surfacing this Olga - I think if we _can_ get an alternative from Wikimedia we should use that instead (the frontend shows that as "png", interestingly). But otherwise, we should probably remove PDFs from the image media set, I don't think most folks would consider PDFs as images 😅

**Comment by obulat** [comment]
> Thanks for surfacing this Olga - I think if we _can_ get an alternative from Wikimedia we should use that instead (the frontend shows that as "png", interestingly). But otherwise, we should probably remove PDFs from the image media set, I don't think most folks would consider PDFs as images 😅 

I know why the frontend "considers" it PNG: we load the "Image not available" placeholder, and replace the image type with the placeholder type 😀 I'll open an issue for that.

@krysal , I didn't notice that it was considered an "image" in Wikimedia, that's interesting! We might need to add a check to the Wikimedia provider script if the pdf images are marked as images in the API responses, and we decide to not accept such pdfs as "image"s. I agree with @AetherUnbound that the users wouldn't expect an image to be a pdf.
